Cargo.toml

```diff
@@ -98,98 +98,95 @@ massa_versioning = { path = "./massa-versioning" }
 massa_wallet = { path = "./massa-wallet" }
 
 # Massa projects dependencies
-massa-proto-rs = {git = "https://github.com/massalabs/massa-proto-rs", "branch" = "main"}
-massa-sc-runtime = {git = "https://github.com/massalabs/massa-sc-runtime", "branch" = "main"}
+massa-proto-rs = {git = "https://github.com/massalabs/massa-proto-rs", "rev" = "cc0b0b3d9b245ac517d28ac6dcf1e87d72b881b7"}
+massa-sc-runtime = {git = "https://github.com/massalabs/massa-sc-runtime", "rev" = "a50d690ec5f7c3d4267e61c6d283e55dff49348f"}
```

Comment:
Not sure what rev. we want, but these commits hashes are not the latest on main.

Reply:
They are now

Reply:
I don't think so for proto-rs.
But cc0b0b is the commit in massa-sc-runtime, so either we update it in both, or we let it like that.
https://github.com/massalabs/massa-proto-rs/commits/main